backend/src/prereq_loader.ts

```diff
@@ -30,20 +30,28 @@ interface NonEmptyQueryResult {
 // prereq query results can be undefined, if the target class doesn't exist.
 type PrereqQueryResult = undefined | NonEmptyQueryResult;
 
+// the curried loader function, expects a termId before producing a loader.
+type CurriedTermIdLoader = (
```

Comment:
I'm not familiar with this terminology. What do you mean by `curried` loader? Also seems like you don't use this type anywhere, so why is it here?

Reply:
Thanks for catching this, I need to remove it. 

Currying is a process of reducing the number of arguments a function requires, producing a function of one less argument. See [here](https://en.wikipedia.org/wiki/Currying).